etcd/src/test/scala/io/buoyant/etcd/KeyTest.scala

```diff
@@ -0,0 +1,444 @@
+package io.buoyant.etcd
+
+import com.twitter.conversions.time._
+import com.twitter.finagle.http._
```

Comment:
I think the Version imported here conflicts with `i.b.etcd.Version`.  Also I think some of these imports are unused.